core/src/test/java/org/infinispan/conflict/impl/ConflictManagerTest.java

```diff
@@ -87,7 +87,7 @@ public void testGetAllVersionsDuringStateTransfer() throws Exception {
       });
       fork(() -> partition(0).merge(partition(1)));
 
-      Map<Address, InternalCacheValue<Object>> versionMap = versionFuture.get(STATE_TRANSFER_DELAY * 8, TimeUnit.MILLISECONDS);
+      Map<Address, InternalCacheValue<Object>> versionMap = versionFuture.get(STATE_TRANSFER_DELAY * 16, TimeUnit.MILLISECONDS);
```

Comment:
Good point. Originally there was some logic to the multiplier, however this is no longer  (maybe never was) applicable. I've  updated the value to be explicitly 60 seconds, which is the same timeout used when waiting for a rebalance.